next.config.js

```diff
@@ -43,6 +43,15 @@ const nextConfig = {
       },
     ];
   },
+  async redirects() {
+    return [
+      {
+        source: '/',
+        destination: '/stake',
+        permanent: true,
```

Comment:
i dont love this being permanent. it really might not be in v3 for example. not to mention it makes development a bit annoying

Reply:
I just don't like next rendering a 404 page on `/`, what do you suggest? A blank `index.ts` page?

Reply:
cant it be permanent false? still a redirect.